desktop/main.js

```diff
@@ -147,8 +147,29 @@ const mainWindow = (() => {
                 titleBarStyle: 'hidden',
             });
 
+            if (!ELECTRON_ENVIRONMENT.isDev()) {
+                const newDotURL = ELECTRON_ENVIRONMENT.isProd() ? 'https://new.expensify.com' : 'https://staging.new.expensify.com';
+
+                // Modify the request origin for requests sent to our API
```

Comment:
Can you please add more to this comment to explain "why?". Currently, it is only explaining "what".